common/src/reserved.rs

```diff
@@ -32,22 +32,17 @@ impl ReservedState {
                     .or_insert(member.consensus_voting_power);
             } else {
                 validator_set
-                    .entry(member.public_key.clone())
+                    .entry(member.name.clone())
                     .and_modify(|v| *v += member.consensus_voting_power)
                     .or_insert(member.consensus_voting_power);
             }
         }
-        let mut validator_set: Vec<(PublicKey, u64)> = validator_set
+        // TODO handle error
```

Comment:
```suggestion
        // TODO: handle error
```
Plus what error handling needed?

Reply:
We do unwrap here.